Source/ToolCore/Build/AndroidProjectGenerator.cpp

```diff
@@ -281,6 +313,55 @@ bool AndroidProjectGenerator::GenerateAndroidManifest()
 
 }
 
+bool AndroidProjectGenerator::CopyUserIcons()
+{
+    FileSystem* fileSystem = GetSubsystem<FileSystem>();
+    ToolSystem* toolSystem = GetSubsystem<ToolSystem>();
+    Project* project = toolSystem->GetProject();
+    AndroidBuildSettings* settings = project->GetBuildSettings()->GetAndroidBuildSettings();
+
+    String userIconPath = settings->GetIconPath();
+    if (!fileSystem->DirExists(userIconPath))               // dont do anything if there is no path defined.
+        return true;
+            
+    String userIconDir = userIconPath + "/drawable";        // 1st target dir 
+    String userIconFile = userIconDir + "/logo_large.png";  // 1st target file
+    String destDir = buildPath_ + "/res/drawable";          // where it should be in the build
+    if ( fileSystem->FileExists (userIconFile) )            // is there a file there?
+    {
+        Poco::File icon1 ( userIconFile.CString() );        // declare the file
+        icon1.copyTo( destDir.CString() );                  // copy the file into the build
```

Comment:
There's a slug of stuff in BuildBase, so probably pass in the Android build instance for the project generator to use, alternatively could refactor this stuff out to BuildSystem, probably opt for the former as it touches less code :) https://github.com/AtomicGameEngine/AtomicGameEngine/blob/master/Source/ToolCore/Build/BuildBase.cpp#L146